codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -135,6 +134,27 @@ public void onPriorityTreeParentChanging(Http2Stream stream, Http2Stream newPare
         });
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Any queued {@link FlowControlled} objects will be sent.
+     */
+    @Override
+    public void channelHandlerContext(ChannelHandlerContext ctx) throws Http2Exception {
```

Comment:
Well at first I just passed in a boolean to the `isWritable` and tracked that.  Then I realized 1 other method (`sendFlowControlled`) that was taking a context as a parameter, inspecting it, saving it, and was previously using it at some other point to do writes.  Then I realized the `FlowControlled` class saves a reference to the context externally and uses that for all its writes.  Seemed more representative to have the flow controller just hang on to 1 explicitly set reference for any flow control activity.